trk/src/Components/SessionItem.js

```diff
@@ -106,64 +106,80 @@ const SessionItem = ({ data, title, renderItem, keyExtractor, isHighlighted}) =>
+                                    <Text style={{ color: 'black', fontSize: 12 }}>{title[1]}</Text>
                                 </View>
-                            </ListItemSessions>
+                                <View style={{ width: '100%', height: '70%', display: 'flex', flexDirection: 'row' }}>
+                                    <View style={{ width: '50%', padding: 10, alignItems: 'center', justifyContent: 'center' }}>
+                                        <Text style={{ color: 'black', fontSize: 32, fontWeight: '500' }}>{data.length}</Text>
+                                        <Text style={{ color: 'black', fontSize: 12 }}>Total Climbs</Text>
+                                    </View>
+                                    <View style={{ width: '50%', justifyContent: 'center', alignItems: 'center' }}>
+                                        <View style={{ marginRight: 10 }}>
+                                            <ListItemSessions dotStyle={styles.climbDot} grade={selectedData[0].grade}>
+                                                <Text style={styles.climbName}>{selectedData[0].name}</Text>
+                                                <View>
+                                                    <Text style={styles.timerInfo}>
+                                                        {timeStampFormatting(selectedData[0].tapTimestamp).replace(/AM|PM/i, '').trim()}
+                                                    </Text>
+                                                </View>
+                                            </ListItemSessions>
+                                        </View>
+                                        <Text style={{ color: 'black', padding: 5, fontSize: 12 }}>Last Climb</Text>
+                                    </View>
+                                </View>
+                            </View>
+                            <View style={{ height: 0.5, backgroundColor: '#BBBBBB', width: '90%', alignSelf: 'center' }} />
                         </View>
-                        <Text style={{color: 'black', padding: 5, fontSize: 12}}>Last Climb</Text>
                     </View>
+
+                </TouchableOpacity>
+                <View style={{ height: 40, flexDirection: 'row', marginTop: -5 }}>
+                    <View style={{ width: '30%' }}>
+                    </View>
+                    <View style={{ width: '70%', flexDirection: 'row', height: '100%' }}>
+                        <View style={{ justifyContent: 'center', width: '60%', alignItems: 'center' }}><Text>PLACEHOLDER</Text></View>
+                        <View style={{ width: 0.5, backgroundColor: '#BBBBBB', alignSelf: 'stretch', marginVertical: 5 }}></View>
+                        <View style={{ justifyContent: 'center', width: '40%', height: '100%', alignItems: 'center' }}><Button title="share" onPress={() => { }}></Button></View>
                     </View>
+
                 </View>
```

Comment:
added this

Reply:
wrapped the initial <TouchableOpacity/> in a <View> to make this addition possible ('column' instead of 'row')